### ISSUE TITLE
CMUS :: Music Player in your terminal

### ISSUE THREAD

**Issue by lalitmee** [iiitv/ShareIt#46]
Have you ever wondered about having an music player which can run from your terminal. **[CMUS](https://github.com/cmus/cmus)** is that music player. 

I think you should give it a try.

**Comment by yashLadha** [comment]
I had tried it, pretty good actually. Can also play videos like audio and
the playlist layout is also good.

**Comment by lalitmee** [comment]
Yeah, it works like vim. If you know some commands and movements of vim. Then you will surely like it. 👍
